client/components/basic/RoomInfo.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { Box, Margins, Icon, Button, ButtonGroup, Divider } from '@rocket.chat/fuselage';
+import { css } from '@rocket.chat/css-in-js';
+
+import RoomAvatar from './avatar/RoomAvatar';
+import { useTranslation } from '../../contexts/TranslationContext';
+import UserCard from './UserCard';
+import VerticalBar from './VerticalBar';
+
+const wordBreak = css`
+	word-break: break-word;
+`;
+
+const Label = (props) => <Box fontScale='p2' color='default' {...props} />;
+const Info = ({ className, ...props }) => <UserCard.Info className={[className, wordBreak]} flexShrink={0} {...props}/>;
+
+const RoomInfoFooter = () => (
```

Comment:
I think it should be part of a possible new component, like 'RoomCard' -> RoomCard.Footer